src/auth/negotiate/kerberos/negotiate_external_acl

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
please use /bin/sh and generic shell code. Use of bash other than for user accounts is considered a security risk these days and we need to avoid it anyway for portability to systems using dash or ksh as their only available shells.